app/controllers/responsiblepeople/SoleProprietorOfAnotherBusinessController.scala

```diff
@@ -74,4 +84,12 @@ class SoleProprietorOfAnotherBusinessController @Inject()(val dataCacheConnector
         }
       }
   }
+
+  def getViewForVat(vatReg: Option[VATRegistered], index: Int, edit: Boolean = false, flow: Option[String] = None) = {
```

Comment:
Refactor into case statement?

Reply:
done

Reply:
Ah i meant to refactor it into the earlier case statement?

Reply:
You need to be more explicit with me:P

Done